src/libraries/System.Security.Cryptography.Pkcs/tests/SignedCms/SignedDocuments.cs

```diff
@@ -1472,5 +1472,159 @@ internal static class SignedDocuments
             "2B78166BDB1612848B535ADC3F0E7BE52991A17F48AFDCCC1698A236BA338930" +
             "50EBAAC4460DAA35185C16670F597E0E6E0CB0AA83F51AAEF452F3367DD9350A" +
             "8A49A5A8F79DF8E921303AB5D6646A482F0F59D9980310E1AE3EE8D77CB857").HexToByteArray();
+
+        internal static readonly byte[] TstWithAttributeCertificate = (
+            "308212E306092A864886F70D010702A08212D4308212D0020103310F300D0609" +
```

Comment:
Your blob is a lot smaller than mine was.  Where'd it come from? :smile:

Reply:
Your blob was an Authenticode signature with a counter signature. This blob is just the counter signature (RFC3161 timestamp).